vector.yaml

```diff
@@ -0,0 +1,22 @@
+sources:
+  validator_logs:
+    type: docker_logs
+    include_containers:
+      - add_validation_node_custom
```

Comment:
We have a few containers we are interested in, not only `add_validation_node_custom`. Also `boot_node`, `cere_archive_node`, `cere_full_node `